### ISSUE TITLE
Google Maps support

### ISSUE THREAD

**Issue by xRealNeon** [alexanderlavrushko/BLE-HUD-navigation-ESP32#20]
Can you add Google Maps support?

**Comment by alexanderlavrushko** [comment]
No, it cannot be done from here. To make it work, the application (in your case, Google Maps) should implement connection to this module, and sending instructions in this specific format. I don't have a contact to Google Maps team, but I'm pretty sure they're not interested.

**Comment by iDoka** [comment]
Would be nice to gathering a list of apps which have capabilities to broadcast navi information over the BLE.

AFAIK, at least Komoot app can send over BLE navigation guidelines: https://github.com/vincent290587/ble_services
